definitions/procedures/dhcp/reservation/add.rb

```diff
@@ -0,0 +1,20 @@
+module Procedures::Dhcp
+  module Reservation
+    class Add < ForemanMaintain::Procedure
+      metadata do
+        description 'add DHCP reservation to subnet'
+        param :subnet, 'specify subnet to add reservation', :required => true
+        param :ip, 'specify IP for new reservation', :required => true
+        param :mac, 'specify mac address for new reservation', :required => true
+        param :name, 'specify name for new reservation', :required => true
+        for_feature :foreman_proxy
+        tags :dhcp_reservations
+        label :dhcp_reservation_add
+      end
+
+      def run
+        feature(:foreman_proxy).add_reservation(@subnet, @ip, @mac, @name)
```

Comment:
There should be a feature called `:dhcp_reservation` with `add/list/remove` methods on it.